### ISSUE TITLE
Brunch cannot include modules using symbolic link

### ISSUE THREAD

**Issue by quangbuule** [brunch/brunch#1564]
Hi,

I think this is the case the Brunch should support:

```bash
cd node_modules
ln -s ../web/static/js app
```

In a module:
```
require('app/lib/myModule');
```

And it failed to load `myModule`:
```
error: Processing of node_modules/app/lib/myModule.js failed. Cannot find module '[...]/package.json'
```

When using node REPL, it's fine to load `app/lib/myModule`

### Environment

1. Brunch: 2.9.1
2. Node: 6
3. NPM: v3
4. Operating system: OSX 10.11

**Comment by denysdovhan** [comment]
Thanks for reaching out!

Seems like this issue might be related to #1593.

**Comment by shvaikalesh** [comment]
Closed in favor of #1593.
